config/SGGame/pokemon.json

```diff
@@ -15841,33 +15841,33 @@
 	"poipole": {
 		"expType": "slow",
 		"num": 803,
-		"rate": "20",
+		"rate": "45",
 		"evolution": {
 			"trigger": null,
 			"evolvesTo": "naganadel"
 		},
-		"baseExp": 114,
+		"baseExp": 189,
 		"evDrops": {
 			"hp": 0,
 			"atk": 0,
 			"def": 0,
 			"spa": 0,
-			"spd": 1,
-			"spe": 0
+			"spd": 0,
+			"spe": 1
```

Comment:
Its 1 spd point, not speed

Reply:
http://prntscr.com/hcflup Actually it is Speed